catalog/core/catalog-core-standardframework/src/main/resources/OSGI-INF/blueprint/blueprint-operators.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- /**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/ -->
+<blueprint
+        xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+        xsi:schemaLocation="http://www.osgi.org/xmlns/blueprint/v1.0.0 http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd">
+
+    <bean id="historian" class="ddf.catalog.history.Historian">
+        <cm:managed-properties persistent-id="ddf.catalog.history.Historian"
+                               update-strategy="container-managed"/>
+        <property name="catalogProviders" ref="catalogProviderSortedList"/>
+        <property name="storageProviders" ref="storageProviderSortedList"/>
+    </bean>
+
+    <bean id="cfOpsSecurity" class="ddf.catalog.impl.operations.OperationsSecuritySupport"/>
+
+    <bean id="cfOpsMetacard" class="ddf.catalog.impl.operations.OperationsMetacardSupport"/>
+
+    <bean id="cfSourceOps" class="ddf.catalog.impl.operations.SourceOperations">
+        <argument ref="frameworkProperties"/>
+    </bean>
+
+    <bean id="cfQueryOps" class="ddf.catalog.impl.operations.QueryOperations">
+        <argument ref="frameworkProperties"/>
+        <argument ref="cfSourceOps"/>
+        <argument ref="cfOpsSecurity"/>
+        <argument ref="cfOpsMetacard"/>
+    </bean>
+
+    <bean id="cfResourceOps" class="ddf.catalog.impl.operations.ResourceOperations">
+        <argument ref="frameworkProperties"/>
+        <argument ref="cfQueryOps"/>
+        <argument ref="cfOpsSecurity"/>
+    </bean>
+
+    <bean id="cfOpsCrud" class="ddf.catalog.impl.operations.OperationsCrudSupport">
```

Comment:
Are you already working on splitting the `*Support` classes into better SRP objects?